src/middlewares/auth.js

```diff
@@ -77,8 +81,13 @@ export default (scopes) => {
           return unauthorized(res);
         }
 
-        req[decoded.scope] = decoded.id;
-        req.authScope = decoded.scope;
+        // XXX s/id/clientId/g
```

Comment:
not sure of this: I think it makes sense that all tokens have the same basic structure. But I can be convinced :)